storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.15.rst

```diff
@@ -0,0 +1,39 @@
+================================================================================
+|Percona XtraBackup| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona XtraBackup| |release| on
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.15/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories. 
+
+All Percona software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- When encrypted tablespaces feature was enabled, encrypted and compressed
+  tables were not usable on the joiner node (Percona XtraDB Cluster) via SST
+  (State Snapshot Transfer) with the ``xtrabackup-v2`` method. Bug fixed
+  :jirabug:`PXB-1867`.
+- In some cases, ``xbcloud`` did not update date related fields of the HTTP
+  header when retrying a request. Bug fixed :jirabug:`PXB-1874`.
+- When the network connection was not stable, ``xbcloud`` could crash after
```

Comment:
408 is not about the network connection. It is returned by google's server when it was unable to complete the request in time. xbcloud did not crash, it just didn't retry the request.